src/components/Form/index.tsx

```diff
@@ -154,7 +158,7 @@ const DVKForm: FunctionComponent<DVKFormProps> = ({
                               errorMessage,
 
                               // default
-                              required, autoFocus, disabled, multiline, autoComplete,
+                              required, autoFocus, disabled, multiline, autoComplete, acceptedFileType, multiple,
```

Comment:
for consistency, `acceptedFileType, multiple` should be placed after a comment `// file`. as you see they are separated by what field uses them